storage/shard.go

```diff
@@ -192,31 +234,33 @@ func (s *dbShard) ReadEncoded(
 	return entry.series.ReadEncoded(ctx, start, end)
 }
 
-func (s *dbShard) series(id string) databaseSeries {
+func (s *dbShard) writableSeries(id string) (databaseSeries, writeCompletionFn) {
 	s.RLock()
-	entry, exists := s.lookup[id]
-	s.RUnlock()
-	if exists {
-		return entry.series
+	if entry, exists := s.lookup[id]; exists {
+		entry.incrementWriterCount()
+		s.RUnlock()
```

Comment:
Gofmt missed this perhaps?  Also shouldn't this be `entry.incrementWriteCount()` and `s.RUnlock()`?